### PR TITLE
docs: Sveltekit tutorial

### LINKED ISSUE #238: SvelteKit Example
I noticed that there are tutorials for the supported frameworks, except for Sveltekit

I would like to contribute a Sveltekit tutorial in the same vein as the existing frameworks.

### PR DESCRIPTION
**Pull request by djamez**
closes #238

### CODE REVIEW

File: docs/tutorials/sveltekit/crud.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
+# CRUD Operations
+
+## Adding new tasks
+
+Now that we can see the list of tasks, it's time to add a few more. We create a form which executes the `addTask` function that invokes `taskRepo.insert()`. Update your `+page.svelte` as follows:
+
+```sveltekit
```
Comment (jycouet) [inline docs/tutorials/sveltekit/crud.md:7]: Could you replace `svelteKit` by `svelte`?
(If not there is an issue with highlights)

File: docs/tutorials/sveltekit/index.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Build a Full-Stack Sveltekit Application
+
+### Create a simple todo app with Remult using a Sveltekit frontend
+
+In this tutorial, we are going to create a simple app to manage a task list. We'll use `Sveltekit` and Remult as our full-stack CRUD framework. For deployment to production, we'll use [Vercel](https://vercel.com/) and a `PostgreSQL` database.
```
Comment (jycouet) [inline docs/tutorials/sveltekit/index.md:5]: Shouldn't we use also [railway.app](https://railway.app/) as it's used in every demo?

**Review by jycouet — COMMENTED** [review]
Thx a lot for the effort.

I quickly went though the tutorial to give an early feedback.

I was quite stuck because I'm missing menus. I think that it's is `docs/.vitepress/config.ts`, to add something like: `{ title: "SvelteKit", path: "sveltekit" },` (You can take it from https://github.com/remult/remult/pull/226/files)

I would be nice to speak about `remultLive.ts`, what do you think?

Actually, I think that it would be nice to pull from https://github.com/remult/remult/pull/226/files in your code, to bring what I started.

Let me know when I can go on with reviewing the PR 👍

**Comment by noam-honig** [comment]
From my perspective, I feel it is important that the tutorials will be
consistent with one another as much as possible - it makes it a lot easier
to maintain.

I understand the urge to deploy to vercel, but it may also adds complexity
- maybe do what I've done with next where I show how to deploy to railway
and then show how to deploy to vercel?
https://remult.dev/tutorials/react-next/deployment.html

**Comment by djamez** [comment]
All the feedback is much appreciated. Let me revise and revert.

**Comment by jycouet** [comment]
Hi there 👋
Any updates on this?

Let me know if you will work on this (when?) or I can finalize your great start?

**Pull request closed without merging**